<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFormulaSetting</class>
 <widget class="QDialog" name="frmFormulaSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmFormulaSetting</string>
  </property>
  <widget class="QTableView" name="tbvwFormula">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>551</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>421</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>200</height>
    </size>
   </property>
   <property name="title">
    <string>气液交换参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>163</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_26">
     <item>
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>F1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sboxF1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>G1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sboxG1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>163</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_31">
     <item>
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>Vf:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sboxVf"/>
     </item>
     <item>
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>Vg:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sboxVg"/>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="sboxF2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>81</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>81</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>F2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>81</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G2:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxG2">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>81</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxF3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxG3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxF4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口4</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxG4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口5</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sboxF5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>46</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sboxG5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>197</width>
     <height>414</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>编号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>药剂名称:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPharName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>母液名称:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLiquorName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>药瓶名称:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPillName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_25">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>西林瓶数量:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxVialNum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_26">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>母液量(ml):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxLiquorVol"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QLabel" name="label_27">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>单瓶母液量(ml):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxSingleLiquorVol"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QLabel" name="label_28">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>目标重量(g):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxTargetWeight"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_34">
      <item>
       <widget class="QLabel" name="label_42">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>摇晃角度(度):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxShakeAngle"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_35">
      <item>
       <widget class="QLabel" name="label_43">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>摇晃次数（次）:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxShakeTimes"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_36">
      <item>
       <widget class="QLabel" name="label_44">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>称重误差(g):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxWeighingError"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_37">
      <item>
       <widget class="QLabel" name="label_45">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>抽气量(ml):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxPumpCapacity"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <widget class="QLabel" name="label_29">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>气液交换次数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sboxExchangeTime"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_32">
    <item>
     <widget class="QPushButton" name="btnNew">
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnModify">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_33">
    <item>
     <widget class="QPushButton" name="btnDel">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
